src/theme/components/notification.ts

```diff
@@ -66,7 +66,7 @@ const Notification = {
       },
       icon: {
         flexShrink: "0",
-        marginEnd: "s",
+        marginEnd: "xs",
```

Comment:
I am curious what's the difference between using paddingStart and marginEnd over paddingLeft, marginRight?

Reply:
`paddingStart`, `paddingEnd`, `marginStart`, and `marginEnd` are all specific to Chakra, whereas `paddingLeft` (padding-left), `paddingRight` (padding-right), `marginLeft` (margin-left), and `marginRight` (margin-right) are all CSS properties (the ones with hyphens are CSS syntax). They are closely related, so `paddingStart` is similar to `paddingLeft` and `marginEnd` is similar to `marginRight`. The reason we are using the "Start" and "End" suffixes is that they help with bidirectionality when the browser's language is set to a right-to-left language.

In English, the "Start" is the left side of any element on the page. In a language such as Arabic, where it is read right-to-left, the "Start" is now considered to be the right side of any element on the page. We want to support right-to-left languages so we use "Start" and "End" instead of "left" and "right".

Reply:
Ahh that's interesting, thank you @EdwinGuzman! So in essence, when dealing with text-related elements, we should always use `start` and `end`, but with spacing between layouts should be free to use any?

Reply:
No, use `Start` and `End` for everything.